assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -118,6 +118,12 @@ systemd:
         WantedBy=multi-user.target
 storage:
   files:
+    - path: /etc/kubernetes/kubeconfig
```

Comment:
This commit implicitly changes how we deliver `kubeconfig` file to controller nodes on AWS without any explanation. This has been changed, as it was causing issues in the past. What is the reasoning behind this change? I think it should be done separately from TLS bootstrapping and well explained.